<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QWidget" name="SettingsDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>357</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>322</width>
    <height>221</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>322</width>
    <height>221</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <widget class="QPushButton" name="Cancel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>320</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OK">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>320</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>571</width>
     <height>301</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QLabel" name="fitModeLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Default fit mode:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="fitModeComboBox">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>ALL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>WIDTH</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NORMAL</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="scalingQualityLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Scaling quality: </string>
     </property>
    </widget>
    <widget class="QComboBox" name="scalingQualityComboBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Better quality</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Better speed</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="fullscreenCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>138</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Open in fullscreen</string>
     </property>
    </widget>
    <widget class="QLabel" name="sortingLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Sorting:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="sortingComboBox">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Name</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Name (desc)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Date</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Date (desc)</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="playVideosCheckBox">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>20</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Webm support (experimental)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="playSoundsCheckBox">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>50</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Play video sounds</string>
     </property>
    </widget>
    <widget class="QLabel" name="ffmpegLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>FFmpeg executable:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ffmpegLineEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>210</y>
       <width>381</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>521</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;FFmpeg is required for webm thumbnail generation. If you are on windows, get it &lt;a href=&quot;http://ffmpeg.zeranoe.com/builds/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#2980b9;&quot;&gt;here&lt;/span&gt;&lt;/a&gt;.&lt;br&gt;Alternative: avconv.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="infiniteScrollingCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Infinite scrolling</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>Loader</string>
    </attribute>
    <widget class="QSlider" name="cacheSlider">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>32</number>
     </property>
     <property name="maximum">
      <number>512</number>
     </property>
     <property name="singleStep">
      <number>2</number>
     </property>
     <property name="value">
      <number>64</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QCheckBox" name="preloaderCheckBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Preloads next image in directory for faster scrolling.</string>
     </property>
     <property name="text">
      <string>Enable preloader</string>
     </property>
    </widget>
    <widget class="QLabel" name="cacheLabel2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="cacheLabel1">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>When preloader used, ram usege may exceed this setting.</string>
     </property>
     <property name="text">
      <string>Cache size, MB:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="reduceRamCheckBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Checking this will make the program keep in memory only currently displayed image (except the preloaded one).</string>
     </property>
     <property name="text">
      <string>Reduce RAM usage</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Appearance</string>
    </attribute>
    <widget class="ClickableLabel" name="bgColorLabel">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="bgColorTextLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Background color:</string>
     </property>
    </widget>
    <widget class="QLabel" name="thumbSizeTextLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Thumbnail size:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="thumbSizeComboBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Small</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Medium</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Large</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Very large</string>
      </property>
     </item>
    </widget>
    <widget class="ClickableLabel" name="accentColorLabel">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="accentColorTextLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Accent color:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="thumbnailLabelsCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>271</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Show thumbnail labels</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="panelPositionLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Panel position:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="panelPositionComboBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>120</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Bottom</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Top</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Left</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Right</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Controls</string>
    </attribute>
    <widget class="QLabel" name="mouseWheelModeLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Mouse wheel mode:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="mouseWheelModeComboBox">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Navigate</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Zoom</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>320</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>customWidgets/clickablelabel.h</header>
   <slots>
    <signal>clicked()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>fitModeComboBox</tabstop>
  <tabstop>scalingQualityComboBox</tabstop>
  <tabstop>sortingComboBox</tabstop>
  <tabstop>fullscreenCheckBox</tabstop>
  <tabstop>OK</tabstop>
  <tabstop>applyButton</tabstop>
  <tabstop>Cancel</tabstop>
  <tabstop>preloaderCheckBox</tabstop>
  <tabstop>reduceRamCheckBox</tabstop>
  <tabstop>cacheSlider</tabstop>
  <tabstop>thumbSizeComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>Cancel</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OK</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>applySettingsAndClose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>applyButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>applySettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cacheSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>cacheLabel2</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bgColorLabel</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>bgColorDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>30</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>accentColorLabel</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accentColorDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>33</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fitModeComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>itemAltered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>417</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fullscreenCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>itemAltered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>455</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>infiniteScrollingCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>itemAltered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>444</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ffmpegLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>itemAltered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>432</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playSoundsCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>itemAltered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>445</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playVideosCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>itemAltered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>487</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scalingQualityComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>itemAltered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>496</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sortingComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>itemAltered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>527</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>preloaderCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>itemAltered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>472</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reduceRamCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>itemAltered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>521</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cacheSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>itemAltered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>552</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>thumbSizeComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>itemAltered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>478</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>panelPositionComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>itemAltered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>536</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>thumbnailLabelsCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>itemAltered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>84</x>
     <y>481</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mouseWheelModeComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>itemAltered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>514</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>applySettingsAndClose()</slot>
  <slot>applySettings()</slot>
  <slot>bgColorDialog()</slot>
  <slot>accentColorDialog()</slot>
  <slot>itemAltered()</slot>
 </slots>
</ui>
